.travis.yml

```diff
@@ -90,4 +90,8 @@ install:
 
 script:
 
+  - >
+    - echo $(black --version);
+    - black --check $(INSTALL_DIR);
```

Comment:
Have not replicated [Iris' line removing .gitignore](https://github.com/SciTools/iris/blob/d16f6767d4d775e2afd691de9c6514782ba2a90d/.travis.yml#L111) - testing has shown that Black is successfully skipping .gitignore automatically.